Content.Server/Medical/CryoPodSystem.cs

```diff
@@ -181,6 +181,9 @@ private void OnActivateUIAttempt(Entity<CryoPodComponent> entity, ref Activatabl
 
     private void OnActivateUI(Entity<CryoPodComponent> entity, ref AfterActivatableUIOpenEvent args)
     {
+        if (!entity.Comp.BodyContainer.ContainedEntity.HasValue)
+            return;
+
         TryComp<TemperatureComponent>(entity.Comp.BodyContainer.ContainedEntity, out var temp);
```

Comment:
Does this break anything if no one's in there.

Reply:
> Does this break anything if no one's in there.

No, it simply doesn't open the window. That's the existing behavior with no occupant.